std/assembly/internal/typedarray.ts

```diff
@@ -205,15 +205,16 @@ export function SOME<TArray extends TypedArray<T>, T>(
   array: TArray,
   callbackfn: (value: T, index: i32, array: TArray) => bool,
 ): bool {
-  var i: i32 = 0;
+
   var length: i32 = array.length;
   var val: T;
-  while (i < length) {
-    val = unchecked(array[i]);
+  var buffer: ArrayBuffer = array.buffer;
+  var byteOffset: i32 = array.byteOffset;
+  for (var i: i32 = 0; i < length; i++) {
```

Comment:
plz remove `i32` from `var i: i32`. By default all int literals which less `2^32-1` using `i32` by default. So this unnecessary. Also this reduce bundle size of assemblyscript which important for browser version.

Reply:
I didn't know adding a type would increase bundle size. I don't want to do that for sure! Let me make the change very quickly.

Reply:
That increase bundle size of assemblyscript because all `std` files includes as ordinal text and cannot be minize

Reply:
btw plz do the same for cached `byteOffset`

Reply:
This should be resolved.

Reply:
Same for `var buffer: ArrayBuffer` which could be inferred

Reply:
```ts
  var buffer = array.buffer;
  var byteOffset = array.byteOffset;
  for (var i = 0; i < length; ++i) {
...
```

Reply:
I will remove all implied types of functions for this pull request. I can also perform these changes on the other functions in this file. May I do this?

Reply:
This should be resolved. Thank you for the really good feedback. I never would have thought of these changes.